src/client/routes/game/GameView.tsx

```diff
@@ -29,8 +29,10 @@ class GameView extends React.Component<IGameViewProps, any> {
       return <div>Loading...</div>
     }
 
-    const currentPlayer = game.currentUser.id === game.playerIdWhite ? Majavashakki.PieceColor.White : Majavashakki.PieceColor.Black
-    const opponentPlayer = game.currentUser.id !== game.playerIdWhite ? Majavashakki.PieceColor.White : Majavashakki.PieceColor.Black
+    const currentPlayer =
+      game.currentUser.id === game.playerIdWhite ? Majavashakki.PieceColor.White : Majavashakki.PieceColor.Black
+    const opponentPlayer =
+      game.currentUser.id !== game.playerIdWhite ? Majavashakki.PieceColor.White : Majavashakki.PieceColor.Black
 
```

Comment:
Tää on mun mielestä ok, vaikka ehkä jonkun mielestä turha rivitys

Reply:
Nää on näitä rivin pituuden rajotuksesta johtuvii hassui

Reply:
132 vs 120 merkkii